src/k8s-frameworkcontroller/deploy/k8s-frameworkcontroller.yaml.template

```diff
@@ -32,7 +32,18 @@ spec:
     spec:
       containers:
       - name: k8s-frameworkcontroller
-        image: frameworkcontroller/frameworkcontroller:v0.3.0
+        image: wang17/frameworkcontroller:v0.4.1
```

Comment:
Please refer to other service

Reply:
Rollback to orignal image until we have a new released FC.